ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/MessageScrubberTest.java

```diff
@@ -704,13 +704,12 @@ public void testUnwantedData1684980GleanUserAgent() {
     assertThrows(UnwantedDataException.class,
         () -> MessageScrubber.scrub(emptyAttributes, Json.createObjectNode()));
```

Comment:
We'll still discard pings with an empty user agent string

Reply:
Ah, empty string. Not a missing string.

Reply:
Yeah, flagging this just in case, it looks to me that it should be discarded.